src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -218,9 +219,11 @@ public void Should_Set_Commandline_Switches()
                 fixture.Settings.OutputFile = "stdout.txt";
                 fixture.Settings.ErrorOutputFile = "stderr.txt";
                 fixture.Settings.Full = true;
-                fixture.Settings.Results = "NewTestResult.xml";
-                fixture.Settings.ResultFormat = "nunit2";
-                fixture.Settings.ResultTransform = "nunit2.xslt";
+                fixture.Settings.Results = new[]
+                {
+                    new NUnit3Result { FileName = "NewTestResult.xml", Format = "nunit2", Transform = "nunit2.xslt" },
```

Comment:
Of course, should someone later make a PR to NUnit to support this, this behaviour may change - so forbidding it in Cake _may_ not be sensible. I'm not sure what Cake's general opinion on such things is.